src/Controls/src/Core/HandlerImpl/Application.Impl.cs

```diff
@@ -107,7 +107,7 @@ public virtual void CloseWindow(Window window)
 		}
 
 		/// <include file="../../../docs/Microsoft.Maui.Controls/Application.xml" path="//Member[@MemberName='ThemeChanged']/Docs" />
-		public void ThemeChanged()
+		void IApplication.ThemeChanged()
```

Comment:
This is just an implementation detail and causes conflicts and ambiguity for user code trying to attach the the very similarly named event.